app/models/validations/sales/financial_validations.rb

```diff
@@ -70,6 +70,27 @@ def validate_percentage_owned_not_too_much_if_older_person(record)
     end
   end
 
+  def validate_equity_in_range_for_year_and_type(record)
+    return unless record.type && record.equity && record.collection_start_year
+
+    if record.collection_start_year == 2022
+      ranges = EQUITY_RANGES_FOR_TYPES_BY_COLLECTION_START_YEAR[2022]
+    elsif record.collection_start_year >= 2023
+      ranges = EQUITY_RANGES_FOR_TYPES_BY_COLLECTION_START_YEAR[2023]
+    end
+
+    if ranges.key?(record.type)
+      range = ranges[record.type]
+      if record.equity < range["min"]
+        record.errors.add :type, I18n.t("validations.financial.equity.under_min", min_equity: range["min"])
+        record.errors.add :equity, I18n.t("validations.financial.equity.under_min", min_equity: range["min"])
+      elsif record.equity > range["max"]
+        record.errors.add :type, I18n.t("validations.financial.equity.over_max")
+        record.errors.add :equity, I18n.t("validations.financial.equity.over_max")
+      end
```

Comment:
```suggestion
      if record.equity < range.min
        record.errors.add :type, I18n.t("validations.financial.equity.under_min", min_equity: range.min)
        record.errors.add :equity, I18n.t("validations.financial.equity.under_min", min_equity: range.min)
      elsif record.equity > range.max
        record.errors.add :type, I18n.t("validations.financial.equity.over_max", max_equity: range.max)
        record.errors.add :equity, I18n.t("validations.financial.equity.over_max", max_equity: range.max)
      end
```

Are you sure that these should be `<` and `>` and not `<=` and `>=`?

Reply:
yep!